<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingScenes</class>
 <widget class="QWidget" name="SettingScenes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>模式設置</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>105</width>
     <height>70</height>
    </rect>
   </property>
   <property name="title">
    <string>模式選擇</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="radioButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>玩家VS電腦</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="text">
       <string>玩家VS玩家</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>確定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_close">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
